### ISSUE TITLE
Consider including secondary GPS data in spec: speed, course, altitude, verticalAccuracy

### ISSUE THREAD

**Issue by ezheidtmann** [openmobilityfoundation/mobility-data-specification#59]
These fields are provided by both dominant smartphone platforms, but they can be null. I believe the field names should be defined in the spec, but allowed to be empty. Proposed definitions:

* `speed`: Instantaneous speed in meters per second. Ref: [iOS](https://developer.apple.com/documentation/corelocation/cllocation/1423798-speed) [Android](https://developer.android.com/reference/android/location/Location#getSpeed())
* `course`: The direction in which the device is traveling, measured in degrees and relative to due north. Ref: [iOS](https://developer.apple.com/documentation/corelocation/cllocation/1423832-course) [Android](https://developer.android.com/reference/android/location/Location#getBearing())
* `altitude`: The altitude, measured in meters. Ref: [iOS](https://developer.apple.com/documentation/corelocation/cllocation/1423820-altitude) [Android](https://developer.android.com/reference/android/location/Location#getAltitude())
* `verticalAccuracy`: The uncertainty of the altitude value, in meters. Ref: [iOS](https://developer.apple.com/documentation/corelocation/cllocation/1423550-verticalaccuracy) [Android](https://developer.android.com/reference/android/location/Location.html#getVerticalAccuracyMeters())

Example point:

```json
    {
        "type": "Feature",
        "properties": {
            "timestamp": 1529968782.421409,
            "speed": 5.23,
            "course": 290,
            "altitude": 40,
            "verticalAccuracy": 30
        },
        "geometry": {
            "type": "Point",
            "coordinates": [
                -118.46710503101347,
                33.9909333514159
            ]
        }
    }
```

**Comment by hunterowens** [comment]
👍 from me. @ezheidtmann do you want to PR? 

@thekaveman this seems like something for a `0.2.0` release?

**Comment by karcass** [comment]
This is covered in Agency 0.3.0.  I don't know if we want to go back and update Provider.

**Comment by karcass** [comment]
What say ye @ezheidtmann ?

**Comment by ezheidtmann** [comment]
I'd advocate for including these fields as optional in Provider feeds.

**Comment by schnuerle** [comment]
Sounds good to add these to provider too. What do you think @thekaveman @karcass?

**Comment by karcass** [comment]
Yes, it would further align Agency and Provider.

**Comment by schnuerle** [comment]
Would anyone like to do a PR for this and make all the fields optional to ensure it's not breaking for 1.1.0? Otherwise we can push to next release. See PR #73 for some ideas.

**Comment by schnuerle** [comment]
I think this may require some more discussion and work and could be a breaking change so moving to 2.0.0.

**Comment by schnuerle** [comment]
Being addressed by #653. Please review that PR and leave comments there if interested.

**Comment by schnuerle** [comment]
We will be addressing most of this in 1.2.0 with the #653 PR. The only thing missing will be vertical accuracy, which we have created a new Issue for at #644.
